mne/bem.py

```diff
@@ -1776,7 +1796,8 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
             skip_symlink = False
```

Comment:
I think there is a logical discrepancy with the handling of ``skip_symlink`` and the logger message, but I'm hesitant to change it because it will change the behavior of the function. Same goes for the lines 1078-1093.